core/src/main/java/org/hibernate/ogm/datastore/impl/DatastoreProviderType.java

```diff
@@ -25,6 +36,8 @@
 	CASSANDRA_EXPERIMENTAL( "org.hibernate.ogm.datastore.cassandra.impl.CassandraDatastoreProvider" ),
 	REDIS_EXPERIMENTAL( "org.hibernate.ogm.datastore.redis.impl.RedisDatastoreProvider" );
 
+	private static final Log log = LoggerFactory.make();
```

Comment:
Why isn't this used below? If there's a reason, maybe it deserves a comment... ?